Button/Button.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+
+const Button = ({onClick,children}) => {
+    return (
+        <button style={{
+            padding:"15px",
+            border: "20px",
+            borderRadius: "50px 50px",
```

Comment:
suppose that these styles are default for every button . but is we want to add more on this styles by sending them as props. how would you do it?

Reply:
Pending